tests/test_integrated_channels/test_sap_success_factors/test_transmitters/test_learner_data.py

```diff
@@ -54,7 +54,8 @@ def setUp(self):
                 error_message='',
                 status=200,
                 enterprise_customer_uuid=self.enterprise_customer.uuid,
-                plugin_configuration_id=self.enterprise_config.id
+                plugin_configuration_id=self.enterprise_config.id,
```

Comment:
I guess we need to change the flow of deployment of these changes.
1- Create field in model is_transmitted and started marking it True when transmission is successful.(This way we will be able to fix the all future instances.)
2- Run the management command. (To fix the existing records and change the is_transmitted when transmission was succesfull. 
3- Now we can add our logic of is_transmitted in the util at the place of 
error_message='', status__lt=400 .